### ISSUE TITLE
Fundraising

### ISSUE THREAD

**Issue by dogles** [HaxeFoundation/Project-Management#27]
(Not sure if I'm supposed to post stuff in here or not.)

There are a lot of companies using Haxe. Some of these companies are big and/or well-funded. Making an effort to get them to donate or become a member of the Foundation would not only increase its budget, but would also increase the visibility and respectability of Haxe.

No one _likes_ to go out and try to raise funds, and I'm not suggesting anyone starts cold-calling asking for money... but I bet it would pay off to take some time to reach out to companies like Autodesk, Nickelodeon, EA, Zynga, etc. and start a relationship with them.

It's a win-win for these companies to invest in the future of Haxe: they're already using it, so it's strategically valuable to help ensure that Haxe remains a thriving platform.

**Comment by larsiusprime** [comment]
Definitely. If they have even a few developers working with Haxe full-time, then they're already spending multiple salaries per year on it, so with the right pitch, a comparatively small investment from them could mean a big difference for the Haxe Foundation.

**Comment by Merelleya** [comment]
I think we should do it and i know I am not the right person to do ot. So @ncannasse ...any thoughts?

I am happy to do the whole "keep in contact and maintain communication", though. I think Hugh put it nicely at WWX: We need a "RAWRRAWR-Guy"

**Comment by Merelleya** [comment]
also: of course you are supposed to post things^^

**Comment by dogles** [comment]
OK! Cool. :)

**Comment by dogles** [comment]
I nominate @larsiusprime, he's got the right messaging. "For only a few cents a day, you could make a big difference in this Foundation's life!" ;)

I'll do some research on how other open source projects go about this sort of thing.

**Comment by ncannasse** [comment]
I'm been doing that since I started HF, from my own experience it varies. In big companies such as Nickelodeon this is actually quite hard to get the right person approval. When the technology is decided by the team, you often need upper management to get involved in signing a HF support contract. Sometimes (Massive if I remember correctly) even if there's a significant internal Haxe usage, the upper management fails to see what it can get back from sponsoring HF, when they don't have much need from support.

OTOH it goes very well when the people which are taking decisions have been involved in the Haxe strategy since the beginning, Like in your company Dan or at TiVo.

I still think we should develop a better "value proposal". Having for instance the capacity to do a joint PR when a new partner join us would help them reach out Haxe community for recruitment/visibility. I think these are important matters for company using Haxe that we should put even more forward.

Maybe we could start by maintaining a list of companies having significant Haxe usage and make sure we are in touch with them on a regular basis, not in order to pester them about joining HF (although this might be necessary to do so from time to time :) ) but also to get the involved in the things we're doing.

For instance I went last year to give a talk+workshop at Gameduell in Berlin while they were still evaluating the technology, I think this kind of things should be multiplied since it will definitely help HF. Maybe I can't do all of these so I might need some help, but I'll always be available if someone has to give the final push to have some new partner joining us.

I was also talking at WWX about HF hiring a business developer, which would visit companies and "sell" them Haxe. That's the next logical step but it's usually a full time job with significant travel expenses so I'm not sure our budget allows it ATM. We still have to keep that in mind for the near future since that would significantly increase our adoption rate.

@Merelleya could you prepare a list of companies with current status / contacts ? (ask me questions if you need more infos), we will then decide on the actions we take on a case by case basis

For other people : feel free to list here companies/contacts you have that might be interested, we will make sure to reach them out.

**Comment by larsiusprime** [comment]
A key issue with fundraising, especially when dealing with enterprise, is to try and figure out how much the person you're talking to is authorized to spend without having to go up the chain for approval. This is why so many software-as-a-service businesses have business models where they charge $499/month (or whatever), rather than asking for $6000 a year. That's because usually the project manager they're selling to is authorized to spend up to $500 (or whatever) on any single thing in a given month and just expense it without having to go higher up for additional approval. Ask for $501 and they need to sign forms and get back to you and then the sale never happens.

Obviously we're a foundation and not a business providing a product/service (at least in the traditional sense), but the same basic principle holds. Instead of asking for big lump sums annually, see if you can take advantage of the self-authorized expense budget, and ask for an amount per month that doesn't require anyone's permission other than the person you're talking to. This does require a little corporate intelligence -- get a feeling for what the typical expense budget is, but we have enough people here involved in businesses we could probably collect that data pretty easily.

**Comment by Merelleya** [comment]
There are a few that I am aware of. I reached out to both Gamesys in London, who are very interested in having actual events (like a local user group) and to Net Entertainment in Sweden. Leo from Net entertainment took some time to send me a fairly detailed E-Mail about what sort of information he would need us to publish in order for him to be able to "sell" Haxe to their Management. What they are mainly looking for are success stories about companies who have switched. They should not be very technical, but rather appeal to Management in that they include data on time/cost saved, faster production, etc. I Was thinking about putting together Material for a small Video Cast (Presentation + Voice?) with some of the details.

So lets see. Where I have spoken to somebody:
- Gamesys (they have one team of 10 people who built one prototype game with Haxe. Early stages
- Net Entertainment (They are evaluating Haxe, but also other technologies)
- Game Duell (We all know about them. They have not brought out a Game with Haxe yet, but their Management wanted them to use it)
- Parts of the BBC (I can ask Dan Hett more about it and see if it makes sense to reach out)
- Autodesk (I emailed Dion yesterday when the Job offer came out, we should stay in contact)

Companies I know of:
- Basically anybody who uses Flambe (Nickelodeon, Disney, and so on)
- EA
- Zynga

Please add to the list.

You may notice, that most of these Companies are involved with games, though.

**Comment by markknol** [comment]
I was thinking of another idea for fundraising. Asking for money at a company is hard unless the full company depends on it. I think there are more haxe developers than companies using Haxe. I think there are quite some developers who want to say "thanks" because they understand what is needed to maintain the full toolkit and they use it for games, software, tools or to hack stuff for fun. 

For these people we only have this lonely button http://haxe.org/foundation/donate.html  
Now I know people want _something_ in return. To confirm you liked their donation. I like what flashdevelop has; [a page with sponsors of the project](http://www.flashdevelop.org/wikidocs/index.php?title=FlashDevelop:Site_support). This takes time to maintain but it somebody does it, it can be really rewarding. If people donate above a certain amount of money, we can feature them as Haxe-rockstar (and give them a star or badge). 

If we can make a deal to get a list of new supporters on regular basis, I'd love to create a nicer page for this on the website. Of course it starts with people who already donated.

---

Other thought:

Since we are in 2016 and we are professional, we could also start a crowdfunding project once a year. Not for WWX but for specific goals, for example; 
- a payed person to do x for certain time;  
- an ad campaign to raise Haxe/macro-awareness;
- another new compiler/library target; 
- utf8/emoji support;
- debugging/refactoring/plugins/tools for editor x; 

In return we can give away early access, shirts, whatever, put people on a lists. I think there are many ways to get attention and raise some money but somebody has to do it some day.

**Comment by ncannasse** [comment]
Great ideas Mark,

We could also have a Patreon, but that usually works better when there's a single individual involved, it's more rare for a team. Also, having technical goals is great, but TBH what we usually don't find is the right people to work on it to the end, instead of the actual budget for it.

Of course with more overall budget we could have a larger team which would cover more skills and be able to work on more different things, but it's a bit hard to commit on working on a specific thing, unless we already know we have plans for it.

**Comment by hereticByte** [comment]
I suggest to develop an **offer for smaller services**. So, a support plan would become more attractive for **freelancers**. 1.200 € per year is still a benchmark.

I also suggest to make special **offers for schools.** They should also be focussed as attractive multipliers. The HAXE workshop (in school) of today will be a **cohort of HAXE programmers** on the job market tomorrow. This makes HAXE (even) more attractive for IT companies. 

If HAXE would be documented a little bit friendlier for programming beginners, it would be ideal to learn the basics for programming. Fast results. Flexible targets. Multiple usecases. I have also some off-topic ideas, how HAXE can attract schools directly.
